docs/getting-started/rulesets.md

```diff
@@ -199,3 +198,141 @@ rules:
 The example above will run the recommended rules from the `spectral:oas2` ruleset, but report `operation-2xx-response` as a warning rather than as an error (as is the default behavior in the `spectral:oas2` ruleset).
 
 Available severity levels are `error`, `warn`, `info`, `hint`, and `off`.
+
+
+### Adding a custom function
+
+If the built-in functions are not enough, Spectral gives you a possibility of providing your own ones.
+Previously, this used to be possible only in case of programmatic usage of Spectral, but there is no such limitation anymore,
```

Comment:
Suggest removing this entire "previously..." sentence - it's not really relevant to anybody and will become stale basically immediately.